medicines/import/src/par.rs

```diff
@@ -47,12 +54,19 @@ pub fn import(dir: &Path, client: Client, mut core: Core, verbosity: i8) -> Resu
                     continue;
                 }
 
-                storage::upload(&hash, &client, &mut core, &file_data, &metadata, verbosity)?;
+                if false == dryrun {
```

Comment:
Is it a bit more readable to put the variable first in the condition, i.e. `if dryrun == false`?

Reply:
It's honestly entirely subjective. Yoda conditions offer a bit of safety in that if you remove one of the `=` here it will entirely fail to compile rather than assigning false to `dryrun`.